### PR TITLE
Add Style

### PR DESCRIPTION
**Pull request by PanRada**
Masonry Style + rebuild structure

### CODE REVIEW

File: src/components/training-list.tsx [hunk 1]
```diff
@@ -12,11 +12,11 @@ export class TrainingList extends React.Component<IProps, {}> {
   render() {
     let trainings = this.props.store.state;
     return (
-      <div>
-        <ul>
+      <div className="wrapper">
```
Comment (piotrwitek) [inline src/components/training-list.tsx]: please reafactor class names using BEM for wrapper, masonry
wrapper -> training-list__container
masonry -> training-list__items
training-list__item
etc.

**Comment by PanRada** [comment]
fixx ready do marge

File: src/app.css [hunk 2]
```diff
@@ -4,12 +4,12 @@ body {
...
 }
 
-.wrapper {
+.traininglist {
   width: 95%;
   margin: 3em auto;
 }
 
-.masonry {
+.traininglist-container {
```
Comment (piotrwitek) [inline src/app.css]: training-list__container - container is element

File: src/components/training-list.tsx [hunk 3]
```diff
@@ -12,8 +12,8 @@ export class TrainingList extends React.Component<IProps, {}> {
   render() {
     let trainings = this.props.store.state;
     return (
-      <div className="wrapper">
-        <div className="masonry">
+      <div className="traininglist">
```
Comment (piotrwitek) [inline src/components/training-list.tsx]: please pay attention to component name to keep class naming consistent
TrainingList -> training-list

File: src/components/training-list.tsx [hunk 4]
```diff
@@ -31,14 +31,14 @@ const TrainingItem = (props: TrainingItemProps) => {
   let handleRemove = () => {
     props.onRemove(training.uid);
   }
-  return <div key={training.uid}  className="item">
-    <div className="item-title">{training.title}</div>
+  return <div key={training.uid}  className="traininglist-item">
```
Comment (piotrwitek) [inline src/components/training-list.tsx]: please pay attention to component name to keep class naming consistent
TrainingItem - training-item

**Comment by piotrwitek** [comment]
@PanRada please make final fixes and should be ready to merge :smiling_imp:

File: src/app.css (hunk 2 above)
Comment (PanRada) [inline src/app.css]: yes mees that,  going to more be strict with BEM metodology
Update: commit pushed

File: src/components/training-list.tsx [hunk 5]
```diff
@@ -12,11 +12,11 @@ export class TrainingList extends React.Component<IProps, {}> {
   render() {
     let trainings = this.props.store.state;
     return (
-      <div>
-        <ul>
+      <div className="training-list">
```
Comment (piotrwitek) [inline src/components/training-list.tsx:15]: very good :+1:

File: src/components/training-list.tsx [hunk 6]
```diff
@@ -31,7 +31,18 @@ const TrainingItem = (props: TrainingItemProps) => {
   let handleRemove = () => {
     props.onRemove(training.uid);
   }
-  return <li key={training.uid} className={''}>
-    {training.title} - <a onClick={handleRemove} href="javascript:undefined">(Remove)</a
-    ></li>
+  return <div key={training.uid}  className="training-list__item">
+    <div className="training-list__item-title">{training.title}</div>
```
Comment (piotrwitek) [inline src/components/training-list.tsx]: not quite :disappointed: it doesn't have anything to do with training-list
TrainingItem is actually a seperate component, it means it's a new block
Should be:
training-item__body
training-item__footer etc.
this way we can move this component to seperate file and you don't need to refactor class names
does make sense?

File: src/components/training-list.tsx [hunk 7]
```diff
@@ -12,11 +12,11 @@ export class TrainingList extends React.Component<IProps, {}> {
   render() {
     let trainings = this.props.store.state;
     return (
-      <div>
-        <ul>
+      <div className="training-list">
+        <div className="training-list__container">
```
Comment (piotrwitek) [inline src/components/training-list.tsx:16]: also good!

File: src/components/training-list.tsx (hunk 6 above)
Comment (PanRada) [inline src/components/training-list.tsx]: not sure it will be easy to move this element from block but fixid on commit

**Comment by PanRada** [comment]
to check

File: src/components/training-list.tsx (hunk 6 above)
Comment (piotrwitek) [inline src/components/training-list.tsx]: of course will be => just copy paste, it is already seperated with proper naming

Comment (piotrwitek) [inline src/components/training-list.tsx]: I'll do it anyway after merge

Comment (PanRada) [inline src/components/training-list.tsx]: is send commit with fix

Comment (piotrwitek) [inline src/components/training-list.tsx]: is not sent

Comment (PanRada) [inline src/components/training-list.tsx]: :100: is now

**Comment by piotrwitek** [comment]
bomba :bomb: :laughing:

**Pull request merged**